src/vs/platform/prompts/common/constants.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { URI } from '../../../base/common/uri.js';
+import { assert } from '../../../base/common/assert.js';
+import { basename } from '../../../base/common/resources.js';
+
+/**
+ * File extension for the reusable prompt files.
+ */
+export const PROMPT_FILE_EXTENSION = '.prompt.md';
+
+/**
+ * Check if provided path is a prompt file.
+ */
+export const isPromptFile = (
+	fileUri: URI,
+): boolean => {
+	return fileUri
+		.path
+		.endsWith(PROMPT_FILE_EXTENSION);
```

Comment:
Does this do what you want:

https://github.com/microsoft/vscode/blob/a948ee5bec485e4dde01abc6d6e289e3f64dbb14/src/vs/base/common/path.ts#L1369

Reply:
Unfortunately this utility returns `.md` instead of `.prompt.md` 🤷